frontend/src/components/comboSection/comboSection.tsx

```diff
@@ -25,26 +25,26 @@ export default class ComboSection extends BaseSection<Message | Artwork | Video
         // TODO: messagecard-center might not used or needed
         if ("messageID" in object) {
             return (
-                <div className="card-section">
+                <div key={id} className="card-section">
```

Comment:
Thanks for fixing these. I kept seeing the console error but smol brain hasn't processed it was coming from here.